addons/binding/org.openhab.binding.vitotronic/ESH-INF/binding/binding.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <binding:binding id="vitotronic"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
-        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+                 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

```

Comment:
This is weird, because we specifically test for tabs in XML files and [all other bindings](https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.airquality/ESH-INF/thing/thing-types.xml) use tabs without any issue.
If you use the standard OH2 IDE and you do an Shift+Ctrl+F (or Source->Format), it will replace all spaces by tabs.